components/badge.css

```diff
@@ -0,0 +1,78 @@
+.badge-container {
+    display: flex;
+    flex-direction: row;
```

Comment:
suggestion: flex-flow: row wrap;  can be used

Reply:
Hey thanks for your suggestion, but I like it to be more descriptive that's why didn't used shorthand.